components/match2/wikis/overwatch/match_group_input_custom.lua

```diff
@@ -271,15 +271,13 @@ function matchFunctions.getVodStuff(match)
 
 	match.links = {}
 	local links = match.links
-	if match.esl then links.esl = 'https://play.eslgaming.com/match/' .. match.esl end
-	if match.owl then links.owl = 'https://overwatchleague.com/en-us/match/' .. match.owl end
-	if match.owc then links.owc = 'https://www.overwatchcontenders.com/match/details/' .. match.owc end
-	if match.jcg then links.jcg = 'http://ow.j-cg.com/compe/view/match/' .. match.jcg end
-	if match.oceow then links.oceow = 'http://bmb.oceoverwatch.com/event/' .. match.oceow end
-	if match.tespa then links.tespa = 'https://compete.tespa.org/tournament/' .. match.tespa end
+	if match.owl then links.owl = 'https://web.archive.org/web/overwatchleague.com/en-us/match/' .. match.owl end
+	if match.jcg then links.jcg = 'https://web.archive.org/web/ow.j-cg.com/compe/view/match/' .. match.jcg end
+	if match.tespa then links.tespa = 'https://web.archive.org/web/compete.tespa.org/tournament/' .. match.tespa end
```

Comment:
not sure if we want to web.archive these links, up to @Rathoz to answer imo

Reply:
iirc the extension was sunset

Reply:
what hjpa said, the extension was sunset because it caused too many false positives. I can see the infoboxes automatically using a webarchive link as fair game.